x11-wm/marco/marco-1.26.1.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MATE2_LA_PUNT="yes"
+
+inherit mate
+
+if [[ ${PV} != 9999 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+fi
```

Comment:
`KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"` (drop the if...fi as live ebuild does not exist anymore)

Reply:
done.